### PR TITLE
adds bloc to keep ui separate from business logic

### PR DESCRIPTION
**Pull request by rlamsal1256**

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -88,13 +109,27 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.8.0+1"
+  provider:
```
Comment (jennyckaplan) [inline pubspec.lock:112]: where do you use this package/what's it for?

File: pubspec.lock [hunk 2]
```diff
@@ -88,13 +109,27 @@ packages:
...
+    source: hosted
+    version: "3.1.0"
   quiver:
     dependency: transitive
     description:
       name: quiver
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.5"
+  rxdart:
```
Comment (jennyckaplan) [inline pubspec.lock:126]: same with this one ?

**Review by jennyckaplan — APPROVED** [review]
LGTM awesome job setting this up

File: pubspec.lock (hunk 1 above)
Comment (rlamsal1256) [reply pubspec.lock:112]: comes as a dependency from flutter_bloc

File: pubspec.lock (hunk 2 above)
Comment (rlamsal1256) [reply pubspec.lock:126]: comes as a dependency from bloc package

**Pull request merged**